data/strings/types_strings.txt

```diff
@@ -10959,21 +10996,65 @@
     zh-Hans = 纪念物
     zh-Hant = 紀念館
 
+  [type.historic.memorial.cross]
+    en = Memorial cross
+    ar = الصليب التذكاري
+    be = Памятны крыж
+    bg = Мемориален кръст
+    cs = Pamětní kříž
+    da = Mindesmærkekors
+    de = Gedenkkreuz
+    el = Σταυρός μνήμης
+    es = Cruz conmemorativa
+    eu = Oroimenezko gurutzea
+    fa = صلیب یادبود
+    fi = Muistoristi
+    fr = Croix commémorative
+    he = צלב זיכרון
+    hu = Emlékkereszt
+    id = Salib peringatan
+    it = Croce commemorativa
+    ja = メモリアルクロス
+    ko = 기념 십자가
+    mr = मेमोरियल क्रॉस
+    nb = Minnekors
+    nl = Herdenkingskruis
+    pl = Krzyż pamiątkowy
```

Comment:
I have no better translation but it sound strange a bit
@lunarna-gh any idea?

Reply:
Not sure if there's a direct translation but more literally it would be `Krzyż pamięci` or, based on the wiki page, `Krzyż pomnikowy`, both fit better than pamiątkowy